app/feeswapper/management/commands/sampleOutput.txt

```diff
@@ -0,0 +1,42 @@
+{'status': '1', 'result': [{'symbol': 'DAI', 'name': 'Dai Stablecoin', 'decimals': '18', 'contractAddress': '0x2448ee2641d78cc42d7ad76498917359d961a783', 'balance': '50000'}, {'symbol': 'OMG', 'name': 'OMGToken', 'decimals': '18', 'contractAddress': '0x879884c3c46a24f56089f3bbbe4d5e38db5788c0', 'balance': '10000000'}, {'symbol': 'WEENUS', 'name': 'Weenus 💪', 'decimals': '18', 'contractAddress': '0xaff4481d10270f50f203e0763e2597776068cbc5', 'balance': '1000000000000000000000'}, {'symbol': 'BAT', 'name': 'Basic Attention Token', 'decimals': '18', 'contractAddress': '0xda5b056cfb861282b4b59d29c9b395bcc238d29b', 'balance': '10000'}, {'symbol': 'MKR', 'name': 'Maker', 'decimals': '18', 'contractAddress': '0xf9ba5210f91d0474bd1e1dcdaec4c58e359aad85', 'balance': '100000000000000000'}], 'message': 'OK'}
```

Comment:
i dont think this needs to go in the codebase.. could just be posted to the PR.

Reply:
Got it, I'll delete in my next commit.